objection/commands/ios/hooking.py

```diff
@@ -319,3 +322,86 @@ def search_method(args: list) -> None:
 
     else:
         click.secho('No methods found')
+
+
+def enumerate(args: list) -> None:
+    """
+        Enumerates the current Android application for classes and methods.
+
+        :param query:
+        :return:
+    """
+    if len(clean_argument_flags(args)) <= 0:
+        click.secho('Usage: ios hooking enumerate \'*[CLASS METHOD]\'', bold=True)
+        return
+
+    shouldPrintOnlyClasses = _should_print_only_classes(args)
+    shouldDumpJSON = _should_dump_json(args)
+    shouldWatchArgs = _should_dump_args(args)
+    shouldWatchRet = _should_dump_return_value(args)
+    shouldBeQuiet = _should_be_quiet(args)
+    shouldBacktrace = _should_dump_backtrace(args)
+
+    api = state_connection.get_api()
+    results = api.ios_hooking_enumerate(args[0])
+
+    # Change the JSON we get into a form that matches the Android enumerate call
+    data = {}
+    for func in results:
+        fullname = func['name']
+        startBracket = fullname.find('[') + 1
+        className = fullname[startBracket: fullname.find(' ')]
+        if data.get(className) is not None:
+            data[className].append(fullname)
+        else:
+            data[className] = []
+
+    # Print the matching methods
+    for klass in data.keys():
+        methods = data[klass]
+        if not shouldBeQuiet:
```

Comment:
I think this will be noisy, as the watcher already prints what it does. I'd suggest we remove it.

Reply:
No longer relevant as I've removed the enumerate methods, the printing now happens in the relevant `search` functions.